task/sast-java-sec-check/0.1/sast-java-sec-check.yaml

```diff
@@ -9,28 +9,30 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs, java"
   name: sast-java-sec-check
 spec:
+  description: >-
+    Static Java code security test.
   workspaces:
     - name: workspace
   results:
-    - description: Test output
+    - description: Tekton task test output.
       name: HACBS_TEST_OUTPUT
   params:
     - name: PATH_CONTEXT
-      description: The path to your source code
+      description: The path to your source code.
       type: string
       default: .
     - name: OUTPUT_FORMAT
-      description: the format of findsecbugs output
+      description: The format of findsecbugs output.
       type: string
       default: "sarif"
     - name: OUTPUT_ONLY_ANALYZE
       description: |
-        only analyze given classes and packages;
-        end with .* to indicate classes in a package, .- to indicate a package prefix
+        Only analyze given classes and packages;
+        end with .* to indicate classes in a package, .- to indicate a package prefix.
       type: string
       default: ""
     - name: OPTIONAL_ARGS
-      description: optional parameters to run findsecbugs
+      description: Optional parameters to run findsecbugs.
```

Comment:
i know that findsecbugs is a tool, but if it's also a command, please put it in monocode (globally): `findsecbugs`